test/mocks/appium/wdio_driver.js

```diff
@@ -49,17 +49,14 @@ class Browser {
       statusBar: { height: 60 },
       navigationBar: { height: 30 }
     });
-    this.execute = jasmine.createSpy().and.callFake((str) => {
-      let res;
-      if (str.includes('getSessionDetails')) {
-        res = {
-          device: deviceName,
-          os_version: 'osVersion',
-          browser_url: 'url'
-        };
-      } else {
-        res = { success: true };
-      }
+    this.execute = jasmine.createSpy().and.callFake(() => {
+      let res = {
+        success: true,
+        deviceName: 'Google Pixel 4',
```

Comment:
this should be `deviceName` as its a param to original function.
 ```suggestion
        deviceName,
```